templates/website/src/blocks/Code/CopyButton/Component.client.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+import { Button } from '@/components/ui/button'
+import Image from 'next/image'
+import { useState } from 'react'
+
+export default function CopyButton({ code }: { code: string }) {
```

Comment:
don't use a default export and instead use named ones for consistency. We're probably not fully consistent here but that's what I'll be aiming for.

In this case since it's one file extra you also don't need to do a nested folder with `Component.client.tsx`, we did that for the block to differentiate the main Component.tsx from its client side

so instead of 

```
import CopyButton from './CopyButton/Component.client'
```

you would use the component like so

```
import { CopyButton } from './CopyButton.tsx'
```

Reply:
Updated to use named export and removed the extra folder.